lighthouse-core/test/config/budget-test.js

```diff
@@ -10,9 +10,9 @@ const assert = require('assert');
 /* eslint-env jest */
 
 describe('Budget', () => {
-  let budget;
+  let input;
```

Comment:
not totally sure what the goal of the rename was here, it's not used in any new tests right?

Reply:
Originally, this file used "budget" to represent the input and "budgets" to represent the result. I thought that was way too subtle and made it easy to use the wrong one.

I changed "input" back to "budget" but kept the "budgets"--> "result" change.

Reply:
gotcha makes total sense :) 👍